src/librustdoc/core.rs

```diff
@@ -235,6 +235,14 @@ pub fn run_core(search_paths: SearchPaths,
         let mut sess = session::build_session_(
             sessopts, cpath, diagnostic_handler, codemap,
         );
+
+        let shutdown_lints = [lint::builtin::UNUSED_IMPORTS,
```

Comment:
wait that's the wrong way around. I thought you were going to turn on the rustdoc lints even if lints are capped

Reply:
I do, but I need to deactivate some lints even if they're enabled.

Reply:
You can just turn the cap lints back on as it was before